src/components/TransformWorkflow/FlowEditor/Components/Canvas.tsx

```
@@ -321,33 +373,62 @@ const getLayoutedElements = ({
-    width: 250,
-    height: 120,
-    marginx: 100,
-    marginy: 100,
-    ranksep: 350,
+  if (nodes.length === 0) return { nodes, edges };
+
+  const spacing = options.spacing || DEFAULT_SPACING;
+
+  // Find connected components using Dagre's graphlib
+  const g = new Dagre.graphlib.Graph();
+  nodes.forEach((n) => g.setNode(n.id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)